src/configuration/syncSettings.js

```diff
@@ -73,5 +80,53 @@ export default function syncSettings(settings) {
         settings.unscheduled_visit_regex = new RegExp(pattern, flags);
     }
 
+    //Define default details.
```

Comment:
Details in this chart are participant-level characteristics, not listing columns.  Filters and measure characteristics don't make sense here.

Reply:
That makes sense. Cleaning it up now.